packages/expo-blur/src/BlurView.types.ts

```diff
@@ -1,5 +1,21 @@
 import { ViewProps } from 'react-native';
 
+/**
+ * Blur method to use on Android.
+ */
+export enum ExperimentalBlurMethod {
+  /**
+   * Falls back to a semi-transparent view instead of rendering a blur effect.
+   */
+  None = 'NONE',
+  /**
+   * Uses a native blur view implementation based on [BlurView](https://github.com/Dimezis/BlurView) library.
+   *
+   * This method may lead to decreased performance and rendering issues during transitions made by `react-native-screens`.
+   */
+  DimezisBlurView = 'DIMEZIS_BLUR_VIEW',
+}
+
```

Comment:
```suggestion
  /**
   *  Blur method to use on Android.
   *
   * - `'none'` - The image is sized to maintain its aspect ratio while filling the container box. If the image's aspect ratio does not match the aspect ratio of its box, then the object will be clipped to fit.
   *
   * - `'demezisBlurView'` - Uses a native blur view implementation based on [BlurView](https://github.com/Dimezis/BlurView) library. This method may lead to decreased performance and rendering issues during transitions made by `react-native-screens.
   *
   * @default 'none'
   */
export type ExperimentalBlurMethod = 'none' | 'dimezisBlurView' 
```

I'd avoid using enums in Typescript.

Reply:
@alanjhughes In this case I prefer enums, I'm aware of some of issues with TypeScript enums, but in my opinion enums will be more usable for the end users, mainly because of IDE autocomplete. 

@tsapeta What do you think?

Reply:
The same autocomplete is available with a union and don't need the extra import. Enums were a mistake of the language and I can't remember the last time I used a library that made use of them. We used them in expo av and people don't like it. More recently in expo image `contentFit` is a type and makes more sense with `Enumerable`s.

Reply:
I agree with Alan, we're rather moving away from using enums on the JS side. Unions should work fine with autocompletion too 🤔